bindings/python/client.py

```diff
@@ -301,9 +301,9 @@ def main():
                     out = '\n'
                     fp = h.file_progress()
                     ti = t.torrent_file
-                    for f, p in zip(ti.files(), fp):
+                    for idx, p in zip(range(ti.num_files()), fp):
```

Comment:
Assuming `len(h.file_progress()) == ti.num_files()` always, then this is simpler:
```python
for idx, p in enumerate(fp):
```